server/parsec/components/postgresql/events.py

```diff
@@ -77,22 +63,29 @@ def _on_notification(conn: object, pid: int, channel: str, payload: object) -> N
                 "Invalid notif received", pid=pid, channel=channel, payload=payload, exc_info=exc
             )
             return
+        logger.info_with_debug_extra(
+            "Received internal event",
```

Comment:
```suggestion
            "Received event",
```

The event is no longer internal since it comes from postgresql ^^